### DISCUSSION TITLE
[Idea] Add a Return method to webservice

### DISCUSSION THREAD

**Discussion by Max84** [PrestaShop/PrestaShop#32245] [Ideas]
### The problem

Orders are available throw webservice, but not product's returns. It can be useful.

### The solution

Add webservice support onto OrderReturn class

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
Sounds good to me 😄
